pandas/tests/test_reshape.py

```diff
@@ -716,6 +716,204 @@ def test_stubs(self):
 
         self.assertEqual(stubs, ['inc', 'edu'])
 
```

Comment:
can you put these in a separate TestWideToLong class (and the existing tests too)